### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by Aurumaker72** [mkdasher/mupen64-rr-lua-#246]
# Rationale

Mupen heavily lacks accessibility, especially for screenreader users.

# Proposal

- New settings category ("Accessibility")
- Optional sounds triggered by commands (frame advance, VCR stop, etc...)
- Announce VCR state (frame count, sample length, etc...) on command

**Comment by Aurumaker72** [comment]
According to user feedback, accessibility deficiencies are mostly found in TASInput.

Moving issue there.
